crm_lead_to_event_registration/__openerp__.py

```diff
@@ -15,9 +15,12 @@
     "depends": [
         'crm',
         'event',
+        'marketing_crm',
```

Comment:
Mixing crm and event installs this automatically, but I decided to put it there to make it clear that it's needed, since I inherit a view from there.